src/abstractdataframe/abstractdataframe.jl

```diff
@@ -950,13 +950,14 @@ Base.hcat(df1::AbstractDataFrame, df2::AbstractDataFrame, dfn::AbstractDataFrame
           makeunique=makeunique, copycols=copycols)
 
 """
-    vcat(dfs::AbstractDataFrame...; cols::Union{Symbol, AbstractVector{Symbol}}=:equal)
+    vcat(dfs::AbstractDataFrame...; cols::Union{Symbol, AbstractVector{Symbol}}=:setequal)
 
 Vertically concatenate `AbstractDataFrame`s.
 
 The `cols` keyword argument determines the columns of the returned data frame:
 
-* `:equal` (the default): require all data frames to have the same column names.
+* `:orderequal`: require all data frames to have the same column names and in the same order.
+* `:setequal`: require all data frames to have the same sets of column names.
```

Comment:
"sets of column names" probably won't be totally obvious for most readers. Maybe add "disregarding order"?

Reply:
fixed